client/lib/wp/sync-handler/index.js

```diff
@@ -197,3 +218,7 @@ export class SyncHandler {
 		return localforage.removeItem( key, fn );
 	}
 }
+
+export const pruneRecordsFrom = () => {
+	cacheInvalidation.pruneRecordsFrom( '2 minutes ago' );
```

Comment:
The PR should have the code you would expect to be committed. It's probably better to add some notes in the description under a `Testing` subhead about temporarily changing the setting to be able to see the invalidation happening more quickly for testing purposes. And actually a "Testing" section in the description would be helpful too.